Jenkinsfile

```diff
@@ -806,7 +806,7 @@ pipeline {
                                 target: 'el8.3',
                                 daos_pkg_version: daosPackagesVersion("centos8", next_version)
                    }
-                } // stage('Test CentOS 7 RPMs')
+                } // stage('Test CentOS 8.3.2011 RPMs') {
```

Comment:
Is this a typo?
```suggestion
                } // stage('Test CentOS 8.3.2011 RPMs')
```

Reply:
Yeah.  It's in a comment, so benign.

Reply:
Ah, true.  Then these could have been comments with approval.